pkg/dashboard/test/server_test.go

```diff
@@ -3478,43 +3477,49 @@ func (suite *miscTestSuite) TestGetFrontendSpec() {
 		"none",
 		"basicAuth"
 	],
-	"supportedAutoScaleMetrics": [
-        {
-            "name": "cpu",
-            "kind": "Resource",
-            "type": "percentage"
-        },
-        {
-            "name": "memory",
-            "kind": "Resource",
-            "type": "percentage"
-        },
-        {
-            "name": "gpu",
-            "kind": "Pods",
-            "type": "percentage"
-        },
-        {
-            "name": "nuclio_processor_stream_high_water_mark_processed_lag",
-            "kind": "External",
-            "type": "int"
-        },
-        {
-            "name": "nuclio_processor_stream_high_water_mark_committed_lag",
-            "kind": "External",
-            "type": "int"
-        },
-        {
-            "name": "nuclio_processor_worker_pending_allocation_current",
-            "kind": "External",
-            "type": "int"
-        },
-        {
-            "name": "nuclio_processor_worker_allocation_wait_duration_ms_sum",
-            "kind": "External",
-            "type": "int"
-        }
-    ]
+	"autoScaleMetrics": {
+		"supportedAutoScaleMetrics": [
```

Comment:
it shows a bit of a redundancy.
perhaps the structure of 
```
{
  "autoScaleMetrics":
    "metricPresets": []
    "windowSizePresets": []
}
```

what do you think?